core/src/main/java/kafka/server/SharePartition.java

```diff
@@ -1107,15 +1123,17 @@ static class InFlightState {
         private InFlightState rollbackState;
         // The timer task for the acquisition lock timeout.
         private TimerTask acquisitionLockTimeoutTask;
+        private String memberId;
```

Comment:
I notice that only three of these fields are rolled back, which I'm sure is correct. I would put them all next to each other in the declarations, and add a comment above the declarations that says just these ones are rolled back.

Reply:
yeah, I have made the change to declare the three fields together and explained about the revert policy in the desciption for `rollbackState`